utilities/rocAL/rocAL_unittests/testAllScripts.sh

```diff
@@ -79,89 +79,84 @@ do
     fi
     for ((rgb=rgb_start;rgb<=rgb_end;rgb++))
     do 
-        # FileSource Reader - Tensor
+        # FileSource Reader
+        ./rocAL_unittests 0 "$image_path" "${output_path}LensCorrection_${rgb_name[$rgb]}_${device_name}" $width $height 45 $device $rgb 0 $display
```

Comment:
Please modify all C++ unit test to support all readers in this PR

Reply:
All changes to C++ examples and unit tests will be covered in another PR